NickvisionCavalier.GNOME/Controls/AddProfileDialog.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using static NickvisionCavalier.Shared.Helpers.Gettext;
+
+namespace NickvisionCavalier.GNOME.Controls;
+
+/// <summary>
+/// A dialog for showing a message
+/// </summary>
+public class AddProfileDialog
+{
+    private readonly Adw.MessageDialog _dialog;
+
+    public string ProfileName;
+    public event EventHandler<MessageDialogResponse>? OnDialogClosed;
```

Comment:
Add documentation of when this event occurs....Add a space above this line to separate it from the Property variables